platform/process/tasks/ABProcessTaskServiceInsertRecord.js

```diff
@@ -46,26 +45,27 @@ module.exports = class InsertRecord extends InsertRecordTaskCore {
                pullDataTasks.push(
                   () =>
                      new Promise((next, bad) => {
-                        let PK = fieldRepeat.datasourceLink.PK();
```

Comment:
Is there a reason you are not using the `req.retry()` here?   

If not, I actually do want you to use the `this._req.retry()` around these calls.  So unless you made an additional change to this call, put this code back.